docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4062,6 +4109,26 @@ supported as follows:
 * If the `index` field is not set, the server returns the configuration for all
   indirect meters in the array identified by the unique id `meter_id`.
 
+### `MeterCounterData`
+
+The `MeterCounterData` P4Runtime message represents the per color counters
+associated with the meter configuration. Whenever a meter is updated and returns
+a color, the corresponding color counter GREEN, YELLOW or RED is updated.
+
+As seen above, these counters can be associated with a `DirectMeterEntry` or
+`MeterEntry`. However, they are implementation specific and targets not capable
```

Comment:
I don't think these counters are "implementation specific". They can either be supported, or not supported, and in both cases this standard defines how they should behave.

Reply:
Removed the implementation specific